src/database/schema/ChatSchema.ts

```diff
@@ -147,7 +146,7 @@ class ChatSchema implements BaseDbSchema {
           ELSE 0 END AS is_continuous
       FROM 
       ${ChatSchema.getTableName()} A 
-      INNER JOIN ${UserSchema.getTableName()} B 
+      LEFT JOIN ${UserSchema.getTableName()} B 
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `INNER JOIN` to `LEFT JOIN` can have significant implications on the result set. An `INNER JOIN` only returns records where there is a match in both tables, while a `LEFT JOIN` returns all records from the left table and matched records from the right table (with NULL in right side columns when there is no match). If this change was intentional and you've considered its impact, then it's fine. Otherwise, you might want to revisit this.



<!-- This is an auto-generated comment by OSS CodeRabbit -->